### PR TITLE
release prep

### PR DESCRIPTION
**Pull request by basz**
@prolic could you look at the composer.json branch aliases if i did that correct?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -40,5 +42,10 @@
     "psr-4": {
       "ZfrOAuth2Test\\Server\\": "tests/src/"
     }
+  },
+  "extra": {
+    "branch-alias": {
+      "dev-master": "0.9.x-dev"
```
Comment (prolic) [inline composer.json]: 0.9-dev

**Comment by prolic** [comment]
This branch has conflicts that must be resolved

**Comment by ojhaujjwal** [comment]
Awesome, Thanks!

**Comment by basz** [comment]
welcome! I would really appreciate it if you could test it and leave some feedback

**Comment by ojhaujjwal** [comment]
Thanks. I will try out this weekend and create issues if I have any queries or feedbacks :D

**Pull request merged**